doc/getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+<!-- TOC -->
+
+- [Getting started](#getting-started)
+    - [SPIRE artifacts](#spire-artifacts)
+    - [Building the artifacts](#building-the-artifacts)
+    - [Installing your artifact in Ubuntu 16.04](#installing-your-artifact-in-ubuntu-1604)
+    - [Configuration](#configuration)
+    - [Running SPIRE with join token](#running-spire-with-join-token)
+
+<!-- /TOC -->
+
+## SPIRE artifacts
+
+You can either download the binaries from [here](https://github.com/spiffe/spire/releases) or build them by
+[setting up your dev environment first](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md).
+
+## Building the artifacts
+
+Note: If you are downloading the artifacts you can skip this step.
+
+In order to build the package you need to do:
+
+    $ make vendor
+    $ make buid
+    $ ./build.sh artifact
+
+This will put the result in the artifacts directory.
+
+## Installing your artifact in Ubuntu 16.04
+
+We will copy your downloaded or generated package into **/opt** and extract the contents in there.
```

Comment:
This is a great section, but am I installing an agent or server?

Reply:
Doing this example on a single VM kind of misses the point of SPIFFE, and won't reflect real world implementations. Suggest showing how it can be used with a single-node server VM and single-node agent VM that connect over a network.

Reply:
:100:

Reply:
I disagree that requiring two nodes is necessary for "getting started" - just complicates the matter, and isn't technically necessary.

Reply:
Something to consider, since we are offering a single tarball with both server and agent every time we install it we are installing both of them, unless we manually select one with its corresponding plugins and configuration files.

Reply:
IMHO at some point we should offer a .deb package that takes care of putting everything in place without users having to move files or set paths manually. An install script is another more generic alternative.